evaluation/utils/io.py

```diff
@@ -5,3 +5,7 @@
 def save_json(content: Dict, path: str, indent: int = 4, **kwargs) -> None:
     with open(path, "w") as f:
         json.dump(content, f, indent=indent, sort_keys=True, **kwargs)
+
+def load_json(path: str) -> Dict:
+    with open(path) as f:
```

Comment:
make it explicit that the file is opened for read-only
```suggestion
    with open(path, "r") as f:
```